region.go

```diff
@@ -47,8 +50,7 @@ type Region struct {
 	Description string
 	VPCRegion   string
 	COSRegion   string
-	Zones       []string
-	SysTypes    []string
+	Zones       map[string]SysTypes
```

Comment:
Not sure we should directly update this Zones data type or add an another filed?

May be if we are planning to update it then once merged lets post this change in channel so consumers are aware of this breaking change.

Reply:
I have checked in CAPI and openshift, this isn't a breaking change.

Reply:
cool, thanks for verifying.